addons/gamification/data/gamification_karma_tracking_demo.xml

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
```

Comment:
Should be noupdate="1"

Reply:
Also remove one indentation level on everyhting.

<data> is at the same level as <odoo> to avoid over-nesting stuff.